src/App.js

```diff
@@ -1,24 +1,15 @@
-import logo from './logo.svg';
-import './App.css';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import HomePage from './pages/homePage/HomePage';
+import DetailsPage from './pages/detailsPage/DetailsPage';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <BrowserRouter>
+      <Routes>
+        <Route path="/" element={<HomePage />} />
+        <Route path="/:id" element={<DetailsPage />} />
+      </Routes>
+    </BrowserRouter>
```

Comment:
**OPTIONAL** - You seem to have built your project to display on only mobile devices, I suggest that to make your app fit across both desktop and mobile screens, you try to make your app responsive. Please fix this 🙏 

Your project:
![trtjtj](https://user-images.githubusercontent.com/73607512/226059375-c2f042da-50f0-4883-b072-f90f97f1d401.PNG)